esx_service/utils/kvESX.py

```diff
@@ -256,32 +257,42 @@ def load(volpath):
     """
     Load and return dictionary from the sidecar
     """
+    logging.info("Load dictionary from sidecar for vol path %s", volpath)
```

Comment:
Mark, the reason I put it as info is the failure cannot hit every time, that is why I want to merge it to master so that every CI run will able to log the enough information that I need. I have more chance to reproduce this issue with needed log.